streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -750,7 +750,7 @@ protected void maybeClean() {
         long now = time.milliseconds();
 
         if (now > lastCleanMs + cleanTimeMs) {
```

Comment:
we only want to run it infrequently, i.e., not every time around the runLoop. I think it should stay